src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -11,13 +12,7 @@
 import com.courtcanva.ccfranchise.services.OrderService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
I think we need to avoid importing packages with `something.*` (aka, wildcard import), if you are using the IDE to handle this, then you might need to check your settings to not avoid this behaviour.

This might help
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports